Assets/Scripts/ThirdPersonPlayer.cs

```diff
@@ -60,36 +77,71 @@ public void OnTriggerEnter(Collider collision)
     private void Update()
     {
         stardustCount.text = "Stardust: " + stardust;
-
-        Debug.Log(stardust);
     }
 
     private void FixedUpdate()
     {
+    
+        if (_levelCleared) return;
+        
         /*
         var xAxisInput = Input.GetAxisRaw("Horizontal");
         var yAxisInput = Input.GetAxisRaw("Vertical");
          */
 
         //InputAction replaces "Input.GetAxis("Example")" and calls function
         //movementInput = inputAction.Player.Move.ReadValue<Vector2>();
+        
+        if (_mapActive) return;
```

Comment:
If we choose to refactor first, then _levelCleared and _mapActive would both be covered by some property _inIso